returnn/tensor/tensor.py

```diff
@@ -123,5 +125,141 @@ def raw_tensor(self, value: Optional[RawTensorType]):
         self._raw_tensor = value
         self.sanity_check(assume_complete=False)
 
+    def _tensor_op_sanity_check(self, other: Union[_frontend_api.RawTensorTypes, Tensor]):
```

Comment:
Remove this function here. This should be directly called in `combine` and `compare`.